README.md

```diff
@@ -98,6 +98,11 @@ protocols:
         exitcode?: number
           # Mocked exitcode of the command.
           # Default: 0
+interpreter: string
+    # The interpreter that should be used to run the tested script. If no
+    # interpreter is specified, a `#!` line at the top of the script will be
+    # used to identify the interpreter.
+    # Example: "/bin/bash", default: ""
```

Comment:
here's where the new interpreter field is documented in `README`. lmk if there are any edits I should make here!